src/utils/code.ts

```diff
@@ -30,6 +30,22 @@ type BuildBlockParams = {
   forceBuildBlock?: boolean
 }
 
+const buildParams = (paramsName: any) => {
```

Comment:
format it, use new lines and indentation.
or create an issue to do it later

Reply:
> format it, use new lines and indentation. or create an issue to do it later

Added an issue for the same - [link](https://github.com/trilogy-group/tu2k22-ui-made-easy/issues/49)